From 82bf036c58c8a133f53752bc682b1ed11b975eb3 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 28 Jul 2017 13:05:39 +0800
Subject: [PATCH 09/13] arm: imx: cpufreq: enable cpu frequency switch on
 ls102x

Freescale-Linux-SDK-for-LS1021A-IOT-Rev2-v0.4-SOURCE-20150907-yocto.iso]

Signed-off-by: Tang Yuantian <Yuantian.Tang@freescale.com>
---
 arch/arm/mach-imx/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 11352f7..978ca8c 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -619,6 +619,7 @@ config FSL_SOC
 config SOC_LS1021A
 	bool "Freescale LS1021A support"
 	select ARM_GIC
+	select ARCH_HAS_CPUFREQ
 	select HAVE_ARM_ARCH_TIMER
 	select FSL_SOC
 	select PCI_DOMAINS if PCI
-- 
1.7.5.4

